composer.json

```diff
@@ -96,7 +96,8 @@
         "takeit/amp-html-bundle": "^0.2.0",
         "asm89/twig-cache-extension": "^1.3",
         "doctrine/doctrine-migrations-bundle": "^1.0",
-        "layershifter/tld-extract": "^1.2"
+        "layershifter/tld-extract": "^1.2",
+        "friendsofsymfony/elastica-bundle": "^4.0"
```

Comment:
it depends how we want to keep that elasticsearch integration, whether it should be part of the publisher project, like other Publisher bundles, or as an extension/plugin which should be installed independently as a 3rd party bundle